app/views/shared/_navbar_user_messages_counter.html.slim

```diff
@@ -0,0 +1,6 @@
+- if current_user.messages.unreaded.count > 0
+  .badge.is-badge-small.is-badge-outlined[data-badge="#{current_user.messages.unreaded.count}"]
+    = current_user.full_name
+- else
+  .badge.is-badge-small.is-badge-outlined[data-badge=nil]

```

Comment:
why all these classes needed if we don't show any numbers? can we just output current_user.full_name?